_docs/development/environment.md

```diff
@@ -9,62 +9,81 @@ menu:
 Your first step toward contributing to Manifold should be to setup a development environment. Manifold requires a number of services for it to run correctly, so getting setup will require a little bit of work.
 
 <div class="notice">
-<strong>Note:</strong> The following steps were followed on a fresh VM running MacOS 10.14 (Mojave). Your mileage may vary on other operating systems.
+<strong>Note:</strong> The following steps were followed on a fresh install of MacOS 10.15 (Catalina). Your mileage may vary on other operating systems.
 </div>
 
 ## Step 1: Install dependencies
 
 ### Install Homebrew and XCode command line tools
 
-[Homebrew](https://brew.sh/) is by far the easiest way to get the underlying dependencies in place for Manifold. Follow the instructions on the [Homebrew Website](https://brew.sh/) to install Homebrew. As of late 2018, the command to install Homebrew is:
+[Homebrew](https://brew.sh/) is by far the easiest way to get the underlying dependencies in place for Manifold. Follow the instructions on the [Homebrew Website](https://brew.sh/) to install Homebrew. As of January 2020, the command to install Homebrew is:
 
 ```
 /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 ```
 
 ### Update your PATH variable
 
-Manifold relies on a number of binstubs. To simplify development, ensure that ./bin and ./node_modules/.bin are both in your $PATH.
+Manifold relies on a number of binstubs. To simplify development, ensure that `./bin` and `./node_modules/.bin` are both in your $PATH.
```

Comment:
I'm not sure how the website will render inline code blocks like this. Is it better to have it like this or the way it was?